src/include/daos_srv/daos_server.h

```diff
@@ -676,7 +676,7 @@ int dsc_obj_list_akey(daos_handle_t oh, daos_epoch_t epoch,
 int dsc_obj_fetch(daos_handle_t oh, daos_epoch_t epoch,
 		daos_key_t *dkey, unsigned int nr,
 		daos_iod_t *iods, d_sg_list_t *sgls,
-		daos_iom_t *maps);
+		daos_iom_t *maps, bool to_leader);
```

Comment:
Hmm, for EC object, it should be able to figure out which shard to fetch by iods, so it does not need add to_leader?

Reply:
I set to_leader to false.  It's the srv_obj_migrate code that needs the DIOF_TO_LEADER flag.

Reply:
I meant to_leader is duplicate with the oh, i.e. you can check this inside dsc_obj_fetch(), 

dsc_obj_fetch() {
      if (!daos_oclass_is_ec(oid))
                flags = DIOF_TO_LEADER;
}

Reply:
Sure, I will make this change.